web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntity.java

```diff
@@ -27,8 +27,8 @@
     @JsonProperty("routes")
```

Comment:
Since this is a breaking change, let's go ahead and add the following based on our latest design:
1. request_id - request id from the user's api call, this is the logging and debugging purpose
2. revision_number - what is the message number for this request, 1, 2, 3, ....n, this is the keep track of the message order
3. message type - DELTA or FULL, this is to handle the out of order/error resource state on the host
4. neighbor_host_dvr_mac - for L3 neighbor, we also need that, we need to figure out who can give that to port manager

Reply:
LGTM